pkg/driver/controller.go

```diff
@@ -336,3 +343,17 @@ func getAccessibilityRequirements(requirement *csi.TopologyRequirement) string {
 	}
 	return ""
 }
+
+// sendEventOrIgnore sends anonymous local-pv provision/delete events
+func sendEventOrIgnore(pvcName, pvName, capacity, stgType, method string) {
+	if utils.GoogleAnalyticsEnabled == "true" {
+		analytics.New().Build().ApplicationBuilder().
+			SetVolumeType(stgType, method).
+			SetDocumentTitle(pvName).
+			SetCampaignName(pvcName).
+			SetLabel(analytics.EventLabelCapacity).
+			SetReplicaCount(analytics.CStorPVReplicaCount, method).
```

Comment:
What happens if somebody has provisioned a volume with replica count 1, for example?
I could not figure out that.

Reply:
This change has been modified , i am getting it from the object directly.